conda/testing/__init__.py

```diff
@@ -240,9 +243,6 @@ def conda_cli(capsys: CaptureFixture) -> CondaCLIFixture:
     """Fixture returning CondaCLIFixture instance."""
     yield CondaCLIFixture(capsys)
 
-    # restore to prior state
-    reset_context()
-
```

Comment:
removed since it didn't actually reset anything, see new `context_aware_monkeypatch` instead